cluster/stone-prd-rh01/tenants/rhtap-build-tenant/external-secrets/kustomization.yaml

```diff
@@ -2,4 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - infra-deployments-pr-creator.yaml
+- registry-pull-secret.yaml
```

Comment:
Can we be more specific about registry? What if we need several registries in the same namespace?

Reply:
Ack, renamed to `registry-redhat-io-pull-secret`. We will be using it for `brew.registry.redhat.io`, but they're the same credentials and come from a registry.redhat.io service account

Reply:
I didn't rename the vault entry (`production/build/rh-syft/registry-pull-secret`) because I don't have permissions to permanently delete stuff, so the old one would stay there anyway